app/views/rubygems/_rubygem.html.erb

```diff
@@ -1,15 +1,25 @@
 <%= link_to rubygem_path(rubygem.name), :class => 'gems__gem' do %>
-  <span class="gems__gem__info">
+  <% if yanked?(rubygem) %>
+    <span class="gems__gem__info">
+    <h2 class="gems__gem__name">
+      <%= rubygem.name %>
+      <span class="gems__gem__badge">Yanked</span>
```

Comment:
i18n

Reply:
Can I reuse `searches.advanced.yanked` here also or should I create a new key under `rubygems` key?

Reply:
yes, reuse is fine.